README.md

```diff
@@ -0,0 +1,120 @@
+# Symbol Statistics Service
+
+It periodically collects information from the network's nodes and produces useful metrics and insights, such as the total number of nodes, their health or their geographical distribution.
+
+## Architecture
+
+The project uses Typescript, Node Express, and MongoDB as database.
+
+```text
+                       +---------------+
+                       |  Application  |
+                       +----+----^-----+
+                            |    |
+                            |    |
+                            |    |  Request / Response
+                            |    |
+ Symbol Statistics Service  |    |
++---------------------------v----+--+
+|                                   |
+| +-Express Server--------+         |
+| |                       |         |
+| |                       |         |
+| |     Task Manager      |         |
+| |                       |         |
+| |                       |         |
+| +-----------+-----------+         |
+|             |                     |
+| +-Services--v-----------------+   |
+| |                             |   |
+| | - Node Monitor              |   |
+| |                             |   |   +------------------+
+| | - Chain Height Monitor      +---+--->                  |
+| |                             |   |   | Mongodb Database |
+| | - Geolocation Monitor       <---+---+                  |
+| |                             |   |   +------------------+
+| +-----------------------------+   |
+|                                   |
++-----------------------------------+
+```
+
+### Node Monitor Task
+
+To discover nodes on the network, a request is made to the catapult-rest endpoint `/node/peer`. This returns the list of peers known to that node, which the task then explores recursively.
+
+To start the process, the task uses the nodes maintained by NEM Group.
+
+Once the whole list of nodes is known, additional information is gathered from each one of them such as `/node/info` and `/chain/info`.
+
+When the search completes, all node information is updated on the `Node` collection.
+
+### Chain Height Monitor Task
+
+To prevent chain forks happening, the finalization height from each node is gathered and height statistics are produced.
```

Comment:
What do you mean? How does the stats service prevent any chain fork?

Reply:
If we found some nodes with the wrong chain height, we can let the node owner resync or update the latest version.
can it consider as prevent?